app/desktop/studio_server/repair_api.py

```
@@ -2,29 +2,28 @@
 
 from fastapi import FastAPI, HTTPException
 from kiln_ai.adapters.adapter_registry import adapter_for_task
+from kiln_ai.adapters.ml_model_list import (
+    default_structured_output_mode_for_model_provider,
+)
 from kiln_ai.adapters.repair.repair_task import RepairTaskRun
 from kiln_ai.datamodel import TaskRun
+from kiln_ai.datamodel.datamodel_enums import StructuredOutputMode
 from kiln_ai.datamodel.json_schema import validate_schema
```

Ruff (0.11.9)

`kiln_ai.datamodel.json_schema.validate_schema` imported but unused

Remove unused import: `kiln_ai.datamodel.json_schema.validate_schema`

(F401)